### PR TITLE
Adding metadata to construct queries

### PR DESCRIPTION
**Pull request by gperonato**
This is the follow up of #274.
The pipelines will now populate the metadata needed to construct the queries.
The SQL view (available as an endpoint in PostgREST) is also updated accordingly to return the parameters.

### CODE REVIEW

File: data-integration/getJRC_GEOPP_DB.py [hunk 1]
```diff
@@ -30,6 +30,19 @@
 UNIT = "MW"
 ISRASTER = False
 NAME = "JRC GEOPP"
+<<<<<<< HEAD
```
Comment (neodark) [inline data-integration/getJRC_GEOPP_DB.py]: Looks like some conflict remaining code left here.
To be removed.

File: data-integration/getJRC_GEOPP_DB.py [hunk 2]
```diff
@@ -30,6 +30,19 @@
...
+=======
```
Comment (neodark) [inline data-integration/getJRC_GEOPP_DB.py]: Looks like some conflict remaining code left here.
To be removed. (as above)

File: data-integration/getJRC_GEOPP_DB.py [hunk 3]
```diff
@@ -30,6 +30,19 @@
...
+# Settings for the query metadata
+# these are the fields that are used to construct a query
+QUERY_FIELDS = []  # empty list means all.
+# these are parameters that added to those automatically generated by the pipeline
+QUERY_PARAMETERS = {
+    "is_tiled": False,
+    "is_raster": False,
+}
+
+>>>>>>> 3b21476... add metadata to JRC_GEOPP_DB
```
Comment (neodark) [inline data-integration/getJRC_GEOPP_DB.py]: Looks like some conflict remaining code left here.
To be removed. (as above)

**Comment by gperonato** [comment]
Hi @neodark, do you have any other comments?

**Comment by neodark** [comment]
Hi @gperonato, not for the moment. 
@Kanma : do you want to review ? Or could we merge this ?

**Comment by PuckCh** [comment]
> Hi @gperonato, not for the moment.
> @Kanma : do you want to review ? Or could we merge this ?

Hi @neodark,
@Kanma has no other comment, so I will merge it.

**Review by PuckCh — APPROVED** [review]

**Pull request merged**